rmcmd.go

```diff
@@ -29,22 +32,23 @@ or all of the containers on a system with the -all flag.`,
 }
 
 func (c *rmcmd) RegisterFlags(fl *flag.FlagSet) {
-	fl.BoolVar(&c.all, "all", false, "Remove all sail containers.")
+	fl.BoolVar(&c.all, "all", false, "Remove all Sail containers.")
+	fl.BoolVar(&c.withData, "with-data", false, "Remove all data associated with the Sail instance.")
```

Comment:
I think this comment should specify that it only removes the cloned project directory